packages/utils/src/functions/bitfield.ts

```diff
@@ -0,0 +1,6 @@
+export const intentsBitfield = (intents: number[] | number) => {
```

Comment:
Better name would be `calculateBitfield()` and use of `bits` instead of `intents` as it is a generic function.